saltgui/index.html

```diff
@@ -200,8 +232,8 @@ <h2 class='time'></h2>
 
       <div class='route' id='page_jobs'>
           <div class='dashboard'>
-            <div id='jobs_page' class='panel jobs-list'>
-              <h1 id="jobs_title">Recent Jobs</h1>
+            <div id='jobs_page' class='panel jobs-list' style="margin-right: 0;">
```

Comment:
style item should move to css file

Reply:
Sure. Will do this. I will fixed also `<p style="display: inline;" id="templatemenuhere">`